pom.xml

```diff
@@ -113,7 +113,7 @@
         <rxjava.version>2.2.21</rxjava.version>
         <commons-codec.verion>1.10</commons-codec.verion>
         <jmh.version>1.29</jmh.version>
-        <awscrt.version>0.19.0</awscrt.version>
```

Comment:
Sorry just noticed this. We should add a changelog entry mentioning the version bump.

Reply:
Added entry for crt bump